perfkitbenchmarker/linux_virtual_machine.py

```diff
@@ -157,6 +161,18 @@ def RobustRemoteCommand(self, command):
                     '--delete']
     return self.RemoteCommand(' '.join(wait_command), should_log=False)
 
+  def SetupRemoteFirewall(self):
+    """Sets up IP table configurations on the VM."""
+    if hasattr(self, 'allowed_ip_range') and self.allowed_ip_range:
```

Comment:
Can we make this more generic so it will work on other providers? Is there any problem with just allowing all traffic (and not using the allowed ip range)? In most cases there should be a firewall outside of the guest anyway...